examples/parcel-hover.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8' />
+    <title>Scoutred Parcel Hover Example</title>
+    <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
+    <script src="https://api.mapbox.com/mapbox-gl-js/v1.11.1/mapbox-gl.js"></script>
+    <link href="https://api.mapbox.com/mapbox-gl-js/v1.11.1/mapbox-gl.css" rel="stylesheet" />
+    <style>
+        body { margin:0; padding:0; }
+        #map { position:absolute; top:0; bottom:0; width:100%; }
+    </style>
+</head>
+<body>
+
+<div id='map'></div>
+<script>
+  const map = new mapboxgl.Map({
+      container: 'map',
+      style: 'https://styles.scoutred.com/v2/scoutred.json',
+      center: [-117.0793,32.7056],
+      zoom: 16
+  });
+  // track the active parcelId
+  var activeParcelId = null;
+
+  map.on('load', function () {
+
+      // When the user moves their mouse over the parcels layer, we'll update the
+      // feature state for the feature under the mouse.
+      map.on("mousemove", "parcels", function(e) {
```

Comment:
In the rest of this file, we are using single-quote strings, but on this line (and 64) we are using double-quotes. I'm open to either option but I think we should be consistent.